packages/messaging/lib/webkit.js

```diff
@@ -393,6 +421,8 @@ function captureGlobals () {
         JSONparse: window.JSON.parse,
         Arrayfrom: window.Array.from,
         Promise: window.Promise,
+        Error: window.Error,
```

Comment:
Why is this platform specific?

Can we move it somewhere else?

Reply:
Ignore it's webkit only. We probably don't want it to be eventually; surely windows needs it?